src/angular/src/app/button-docs/button-docs.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-button-docs',
+  templateUrl: './button-docs.component.html',
```

Comment:
Thoughts on using template instead of templateUrl? I think having the html in the component ts file would make development easier (to associate the ts with the html) and cut down on the number of files Spark has to have.